python/cudf/cudf/core/dataframe.py

```diff
@@ -6409,7 +6409,20 @@ def count(self, axis=0, numeric_only=False):
             raise NotImplementedError("Only axis=0 is currently supported.")
         length = len(self)
         return Series._from_column(
-            as_column([length - col.null_count for col in self._columns]),
+            as_column(
+                [
+                    length
+                    - (
+                        col.null_count
+                        + (
+                            col.nan_count
```

Comment:
I think this will currently fail if `count` is called on a non-numeric column. If https://github.com/rapidsai/cudf/pull/17765 is merged, then this will be valid

Reply:
`nan_count` for all other columns is a `0` no?

Reply:
ah, my bad. I was mixing up looking into polars code.

Reply:
Fixed it.